docs/Operators.md

```diff
@@ -136,6 +136,7 @@
   * <a href="#Size">Size</a>
   * <a href="#Slice">Slice</a>
   * <a href="#Softmax">Softmax</a>
+  * <a href="#SoftmaxCrossEntropyLoss">SoftmaxCrossEntropyLoss</a>
```

Comment:
>SoftmaxCrossEntropyLoss [](start = 14, length = 23)

 nit. 
how about just call it SoftmaxCrossEntropy

Reply:
ah... I see WeiSheng's comment ... 

---
In reply to: [377937804](https://github.com/onnx/onnx/pull/2573#discussion_r377937804) [](ancestors = 377937804)

Reply:
If it's already universally called *Loss, I am ok with the current naming

---
In reply to: [377938280](https://github.com/onnx/onnx/pull/2573#discussion_r377938280) [](ancestors = 377938280,377937804)